mock/pkg/cluster/manager/manager.go

```diff
@@ -207,18 +207,3 @@ func (mr *MockManagerMockRecorder) UpdateByID(ctx, id, cluster interface{}) *gom
 	mr.mock.ctrl.T.Helper()
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "UpdateByID", reflect.TypeOf((*MockManager)(nil).UpdateByID), ctx, id, cluster)
 }
-
-// UpdateTemplateByID mocks base method.
-func (m *MockManager) UpdateTemplateByID(ctx context.Context, id uint, template, release string) (*models.Cluster, error) {
```

Comment:
Is this method not necessary any more ?

Reply:
yeah, this method is removed in previous commit.